packages/web-components/src/components/va-alert-expandable/va-alert-expandable.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Prop,
+  Listen,
+  State,
+  h,
+} from '@stencil/core';
+import classnames from 'classnames';
+
+@Component({
+  tag: 'va-alert-expandable',
+  styleUrl: 'va-alert-expandable.css',
+  shadow: true,
+})
+export class VaAlertExpandable {
+  @Element() el: HTMLElement;
+
+  @State() open: boolean;
+
+  /**
+   * Determines the icon and background color.
+   * One of `info`, `error`, `success`, `warning`, or `continue`
+   */
+   @Prop() status?: 'continue' | 'error' | 'info' | 'success' | 'warning'; 
+
+  /**
+   * The text to trigger the expansion
+   */
+  @Prop() trigger: string = '';
+
+  /**
+   * If true, doesn't fire the CustomEvent which can be used for analytics tracking.
+   */
+  @Prop() disableAnalytics: boolean = false;
+
+  /**
+   * If false, the status icon is removed.
+   */
+  @Prop() showIcon: boolean = true;
```

Comment:
I think this default should be flipped. We want people to be using the components like this:

```html
<va-alert-expandable show-icon /> <!-- With icon -->
<va-alert-expandable /> <!-- Without icon -->
```

Rather than:

```html
<va-alert-expandable /> <!-- With icon -->
<va-alert-expandable show-icon="false" /> <!-- Without icon -->
```

This is about what is considered a [valid HTML boolean attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#boolean_attributes):

> To be clear, the values "true" and "false" are not allowed on boolean attributes. To represent a false value, the attribute has to be omitted altogether.

If the icon should be part of the default look, then the prop name can change to reflect the default falsey behavior, like:

```html
<va-alert-expandable /> <!-- With icon -->
<va-alert-expandable iconless /> <!-- Without icon -->
<!-- Where `iconless` or whatever the prop is named is defaulted to false -->
```